src/index.js

```diff
@@ -106,6 +135,25 @@ function generateForm(formData) {
       case "text":
         const [textElement, label] = generateTextInputAndLabel(fieldData);
         formElement.append(label, textElement);
+        break;
+      case "radio":
+        const radioInputLabelElement = document.createElement("label");
+        radioInputLabelElement.htmlFor = `${fieldData.id}Radio`;
+        radioInputLabelElement.innerText = fieldData.label;
+        formElement.append(radioInputLabelElement);
+        const divElement = document.createElement("div");
+        divElement.id = `${fieldData.id}Radio`;
+        formElement.append(divElement);
+        let x = 0;
+        fieldData.options.forEach((optionData) => {
+          const [radioElement, radioLabel] = generateRadioInputAndLabel(
+            fieldData,
+            x
+          );
+          divElement.append(radioElement, radioLabel);
+          x++;
+        });
```

Comment:
Here `optionData` is actually equivalent to `fieldData.options[x]`, because the variable you get from `forEach` is actually the thing itself at each iteration. It's like a shorthand for the above notation. so you can do this instead: 

```suggestion
        fieldData.options.forEach((optionData) => {
          const [radioElement, radioLabel] = generateRadioInputAndLabel(
            fieldData,
            optionData
          );
          divElement.append(radioElement, radioLabel);
        });
```